service/turn.rest/src/main/java/org/kaazing/gateway/service/turn/rest/DefaultCredentialsGenerator.java

```diff
@@ -0,0 +1,96 @@
+/**
+ * Copyright 2007-2016, Kaazing Corporation. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.kaazing.gateway.service.turn.rest;
+
+import java.time.Instant;
+import java.util.Base64;
+
+import javax.crypto.Mac;
+import javax.crypto.spec.SecretKeySpec;
+import javax.security.auth.Subject;
+
+/**
+ * Default implementation of the TurnRestCredentialsGenerator
+ * It contains a standard implementation of the short term credentials 
+ * as described in <a href='https://tools.ietf.org/html/draft-uberti-behave-turn-rest-00'>draft-uberti-behave-turn-rest-00</a> 
+ */
+public class DefaultCredentialsGenerator implements TurnRestCredentialsGenerator {
+
+    public static final String HMAC_SHA1_ALGORITHM = "HmacSHA1";
```

Comment:
Please document the Keytool commands to set that up in the Javadoc of this class.  Thanks